install_files/ansible-base/roles/tails-config/templates/extension.js.in

```diff
@@ -91,6 +91,11 @@ class Indicator extends PanelMenu.Button {
         keypass.connect('activate', () => {
             Util.trySpawnCommandLine(`keepassxc`);
         });
+
+        let filebrowser = new PopupMenu.PopupMenuItem(_('Open File Browser'));
```

Comment:
Is this overkill? there's already a Places menu

Reply:
@zenmonkeykstop One of the issues I've observed in various trainings and in discussions with the DST team is that figuring out how to even open the file browser can be challenging for new users. They don't associate the Places menu with browsing files, so they go to the Activities menu. Sometimes they can get to Nautilus quickly by searching for 'Files', but often they'll manually hunt around the various categories until they find an icon with a folder on it. By putting this option in the extension, it eliminates the confusion from Places, and the challenge of hunting around the Activities view for it.

Reply:
OK, but it also introduces a non-SD related menu item to a menu labelled SecureDrop, and as currently implemented it opens the Persistent directory, when we recommend that journalists default to using non-persistent directories instead (see https://docs.securedrop.org/en/stable/journalist.html#step-1-download-the-encrypted-submission). I don't know if we recommend use of the Persistent directory (`Persistent/Tor Browser`, yes) anywhere in the docs for submissions. There's also further opportunity for new user confusion here, as the first thing they'll see on using this option is a directory called `securedrop`, with which they should definitely not be messing.

The original intent of the PR as I understood it was to get our UX additions more in line with the Tails UX. This feels kindof counter to that intent (we're now working around perceived issues with it?) and a bit scope-creepy.

Reply:
This is still in place but not a blocker for merge.